### PR TITLE
CGStructure updates

### PR DESCRIPTION
**Pull request by GearsAD**
Updates the functions required to persist the solver data via CGDFG

### CODE REVIEW

File: src/CloudGraphsDFG/entities/CGStructure.jl [hunk 1]
```diff
@@ -10,17 +10,34 @@ mutable struct User <: AbstractCGNode
     id::Symbol
     name::String
     description::String
-    # labels::Vector{Symbol}
     data::Dict{Symbol, String}
+    createdTimestamp::String
+    lastUpdatedTimestamp::String
+    User(id::Symbol,
```
Comment (Affie) [inline src/CloudGraphsDFG/entities/CGStructure.jl:16]: https://docs.julialang.org/en/v1/manual/constructors/#man-inner-constructor-methods-1

> It is good practice to provide as few inner constructor methods as possible: only those taking all arguments explicitly and enforcing essential error checking and transformation. Additional convenience constructor methods, supplying default values or auxiliary transformations, should be provided as outer constructors that call the inner constructors to do the heavy lifting. This separation is typically quite natural.

**Comment by Affie** [comment]
Do you only want this in v0.8.0?

File: src/CloudGraphsDFG/entities/CGStructure.jl (hunk 1 above)
Comment (GearsAD) [reply src/CloudGraphsDFG/entities/CGStructure.jl:16]: Huh, good note! Thanks, I'll move it outside.

**Comment by GearsAD** [comment]
This is now merged to #417

**Pull request closed without merging**